packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -374,12 +360,12 @@ abstract class BaseMethod implements PythonBase {
                 for (const prop of liftedProperties) {
                     const paramName = toPythonParameterName(prop.name);
                     const paramType = resolver.resolve(prop.type, { forwardReferences: false });
-                    const paramDefault = prop.type.optional ? "=None" : "";
+                    const paramDefault = prop.type.nullable ? "=None" : "";
```

Comment:
This is not doing method arguments, right? A non-optional nullable type shouldn't have a default.